models/intermediate/int_klaviyo__person_metrics.sql

```diff
@@ -0,0 +1,39 @@
+with person_campaign_flow as (
+
+    select *
+    from {{ ref('klaviyo__person_campaign_flow') }}
+),
+
+{%- set pcf_columns = adapter.get_columns_in_relation(ref('klaviyo__person_campaign_flow')) %}
+
+agg_metrics as (
+
+    select
+        person_id,
+        count(distinct last_touch_campaign_id) as count_total_campaigns,
+        count(distinct last_touch_flow_id) as count_total_flows,
+        min(first_touch_at) as first_touch_at, -- first ever event occurred at
```

Comment:
This includes all event types, right? I would stay away from calling these first_touch_at and last_touch_at if they aren't truly related to attribution